.travis.yml

```diff
@@ -27,7 +27,7 @@ script:
       (
         set -x;
         cargo test --verbose --release &&
-        cargo test --verbose --release --features use-mock-crypto
+        cargo test --verbose --release --features mock
```

Comment:
I think we can add a line similar to https://github.com/Fraser999/maidsafe_utilities/blob/master/.travis.yml#L34 to test the seeded rng in a standalone test run.